e2e-tests/cypress/tests/integration/channels/search/clear_input_spec.js

```diff
@@ -52,13 +55,8 @@ describe('Search', () => {
 
         // * Verify search input field exists and not search button, as inputs contains placeholder not buttons/icons
         // and then type in a search text
-        cy.uiGetSearchBox().click().wait(TIMEOUTS.HALF_SEC).type(searchText + '{enter}');
-
-        // # Click on the pinned post button from the header
-        cy.uiGetChannelPinButton().click();
-
-        // * Verify the pinned post RHS is open
-        cy.uiGetRHS().should('contain', 'Pinned messages');
+        cy.uiGetSearchContainer().click();
+        cy.uiGetSearchBox().clear().wait(TIMEOUTS.HALF_SEC).type(searchText + '{enter}').wait(TIMEOUTS.HALF_SEC);
```

Comment:
This still failed with:
```
This element `<input#sbrSearchBox.search-bar.form-control.a11y__region>` is not visible because its parent `<div.search-bar__container.channel-header.alt>` has CSS property: `display: none`
    at Context.eval (webpack://cypress/./tests/support/ui/global_header.js:110:33)
```